### ISSUE TITLE
ClosePath does not move the Cursor

### ISSUE THREAD

**Issue by flippmoke** [mapbox/vector-tile-spec#49]
I am creating this issue for open discussion and a note for future releases. Currently the `ClosePath` command does not move the cursor. This was not described in v1 and most existing vector tiles are currently encoded in such a way that `ClosePath` does not move the cursor. From a logical standpoint, it might make sense to have `ClosePath` move the cursor to the initial point of that linear ring.

**Comment by fosskers** [comment]
Yeah, codecs have to work around that and you could consider it a "gotcha" in the spec.

**Comment by mapsam** [comment]
@fosskers this is documented (albiet quite deeply) in the interactive spec page here: https://www.mapbox.com/vector-tiles/specification/#encoding-geom

Check out `step 5` - which doesn't move the "pen".

**Comment by fosskers** [comment]
@mapsam Yeah, I made sure not to miss that when writing my codecs. By "gotcha" I meant it adds a bit of mental overhead from being unintuitive behaviour.
